### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by ukqfhf** [VKCOM/VKUI#5975] [Ideas] [answered]
На сайте нет раздела Roadmap, а хочется понимать что сейчас находится в разработке и планируется к реализации. 

Референсы:

* [roadmap MUI](https://mui.com/material-ui/discover-more/roadmap/)

**Comment by BlackySoul** [comment, marked as answer]
Привет! Мы используем систему [milestones](https://github.com/VKCOM/VKUI/milestones), в целом, там можно увидеть, что находится в текущей разработке и какие планы. 
Вот ещё публичная [доска](https://github.com/orgs/VKCOM/projects/5/views/10) с задачами следующего мажорного релиза

Но соглашусь, что формат может быть не особо `friendly`, если хочется просто оценить ключевые изменения. Подумаем, как можно будет улучшить этот момент, спасибо за фидбэк х)
